GitHub Connector documentation update (#95)
kyma-incubator/hack-showcase: docs/github-connector/configuration.md

# Configuring GitHub Connector

## Overview

> This document describes how to correctly connect GitHub repository to the GitHub Connector installed in Kyma environment. After completion you will be able to handle events incoming from GitHub in lambdas.

Reviewer: As per guidelines, don't use future tense. See [here](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#tenses).

## Prerequisites

- Kyma with the GitHub Connector [installed](/docs/github-connector/installation.md)
- Connection to Kyma

## Installation

> 1. Find the newly created github-connector application and [bind it to the namespace](https://kyma-project.io/docs/components/application-connector/#tutorials-bind-an-application-to-a-namespace) of your choice.

Reviewer: Is `github-connector` the name of the app (file)? Or should it be `The GitHub Connector` application? Change if necessary.

> 2. Open settings of GitHub repository you want to connect to, go to `Webhooks` page and click `Add webhook`.

Reviewer: ```suggestion
2. Open the settings of the GitHub repository you want to connect to, go to `Webhooks` page and click `Add webhook`.
```

> 3. On the configuration page, fill the field `Payload URL` with exposed service URL (you can find it in `Kyma Console > {NAMESPACE} > APIs`) and add `/webhook` at the end of it.

Reviewer: ```suggestion
3. On the configuration page, fill the field `Payload URL` with exposed service URL (you can find it in `Kyma Console > {NAMESPACE} > APIs`) and add `/webhook` at the end of the URL.
```

4. Set other fields as follows:

    - **Content type**: `application/json`
    - **Secret**: `my-secret-key`
    - **SSL verification**: `Disabled`

>     >**NOTE:** Secret is temporarily defined statically in code and SSL verification is disabled. It will be changed in the future.

Reviewer: As per guidelines:
> **NOTE:** Do not use words such as "currently" or "now" to indicate that something is in the transitional phase of development. Avoid promising anything and mention only those components and functionalities that are already in use.

See [this](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology).

> 5. Select which events you would like to receive in GitHub Connector.

Reviewer: ```suggestion
5. Select which events you want to receive in GitHub Connector.
```

Is it `GitHub Connector` or `The GitHub Connector`? You use both in different files. Please check, adjust and be consistent.

> 6. Click `Add webhook`. You will be redirected back to webhooks page. There will be new webhook on list. Successful configuration results with green tick next to it.

Reviewer: ```suggestion
6. Click `Add webhook`. This redirects you back to the webhooks page. You can see a new webhook in the list. A successful configuration results in a green tick next to the new webhook.
```
